### PR TITLE
Fixed the XML exporters for the statistics

### PR DESCRIPTION
**Pull request by micheles**
Closes https://github.com/gem/oq-risklib/issues/678 and https://github.com/gem/oq-risklib/issues/662

### CODE REVIEW

**Comment by ptormene** [comment]
Please update also the changelog

**Comment by CatalinaYepes** [comment]
The exporters are working for `agg_curve-stats`, `avg_losses-stats`, `loss_curves-stats` and `loss_maps-stats`.

It is still missing `agg_losses-stats`

File: openquake/commonlib/export/risk.py [hunk 1]
```diff
@@ -641,30 +669,37 @@ def _gen_writers(dstore, writercls, root):
...
+                            loss_type=loss_type, unit=ct['unit'],
+                            **get_paths(rlz)), (
+                                loss_type, poe, rlz.ordinal, ins)
+                elif root.endswith('-stats'):
+                    pairs = [('mean', None)] + [
+                        ('quantile-%s' % q, q)
+                        for q in oq.quantile_loss_curves]
+                    for ordinal, (statname, statvalue) in enumerate(pairs):
+                        dest = dstore.export_path('%s-%s-%s%s%s.xml' % (
+                            root[:-6], statname, loss_type, poe_str,
```
Comment (ptormene) [inline openquake/commonlib/export/risk.py]: Please add a comment

File: openquake/commonlib/export/risk.py [hunk 2]
```diff
@@ -641,30 +669,37 @@ def _gen_writers(dstore, writercls, root):
...
+        poe_str = '-%s' % poe if poe is not None else ''
+        for l, ct in enumerate(cost_types):
+            loss_type = ct['name']
+            for ins in range(oq.insured_losses + 1):
+                if root.endswith('-rlzs'):
+                    for rlz in rlzs:
+                        suffix = ('' if L == 1 and R == 1
+                                  else '-gsimltp_%s_%s' % (rlz.uid, loss_type))
+                        dest = dstore.export_path('%s%s%s%s.xml' % (
+                            root[:-5], suffix, poe_str, '_ins' if ins else ''))
```
Comment (ptormene) [inline openquake/commonlib/export/risk.py]: Please add a comment

File: openquake/commonlib/export/risk.py (hunk 1 above)
Comment (micheles) [inline openquake/commonlib/export/risk.py]: Done

File: openquake/commonlib/export/risk.py (hunk 2 above)
Comment (micheles) [inline openquake/commonlib/export/risk.py]: Done

**Comment by ptormene** [comment]
LGTM

**Pull request merged**